sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuShuffleEnv.scala

```diff
@@ -146,4 +146,14 @@ object GpuShuffleEnv extends Logging {
   def getDeviceStorage: RapidsDeviceMemoryStore = env.getDeviceStorage
 
   def rapidsShuffleCodec: Option[TableCompressionCodec] = env.rapidsShuffleCodec
+
+  def shuffleFetchTimeoutSeconds: Long = {
+    val sparkEnv = SparkEnv.get
+    val defaultValue = 120
+    if (sparkEnv != null) {
```

Comment:
Are we risking this getting called outside of a Spark context and silently not honoring the config setting?

Reply:
Yeah that would be bad, especially if you call this thinking you'll get the actual number, and then make decisions on it.

Sigh.. I'll think of something else. Also, I am OK moving this to branch-0.3 btw, I do not think we have to have this in branch-0.2 at this point.

Reply:
Ok, I am passing this as a parameter now (https://github.com/NVIDIA/spark-rapids/pull/732/commits/37651a9082c8bc8f4b4ac1731a3b6630edd155a4). It will blow up if the shuffle env isn't setup correctly, just like the other settings would. This is better than my prior impl. Thanks for bringing it up.